build/media-suite_compile.sh

```diff
@@ -2465,6 +2486,10 @@ if [[ $ffmpeg != no ]]; then
 
         enabled libvvdec && grep_and_sed FF_PROFILE libavcodec/libvvdec.c 's/FF_PROFILE/AV_PROFILE/g'
 
+        # Bypass ffmpeg check for audiotoolbox
+        enabled audiotoolbox && do_addOption --extra-libs=-lAudioToolboxWrapper && do_addOption --disable-outdev=audiotoolbox &&
+            sed -ri "s/check_apple_framework AudioToolbox/check_apple_framework/g" /build/ffmpeg-git/configure
```

Comment:
```suggestion
        enabled audiotoolbox && do_addOption --extra-libs=-lAudioToolboxWrapper && do_addOption --disable-outdev=audiotoolbox &&
            sed -ri "s/check_apple_framework AudioToolbox/check_apple_framework/g" configure
```

You should already be in `/build/ffmpeg-git` by now, so this full path will only induce confusion.